integration/simulation/validate_chain.go

```diff
@@ -35,6 +35,9 @@ func checkNetworkValidity(t *testing.T, s *Simulation) {
 // - noReorgs
 func checkEthereumBlockchainValidity(t *testing.T, s *Simulation) uint64 {
 	// Sanity check number for a minimum height
+	// TODO review the minHeight
```

Comment:
these need to be removed once the block time gets sorted

Reply:
Done! :)